Sources/ArcGISToolkit/Components/Bookmarks/Bookmarks.swift

```diff
@@ -173,46 +222,35 @@ extension Bookmarks {
         )
     }
     
-    /// A view that shows the list of bookmarks.
-    ///
-    /// - Note: Once the minimum supported platform is 16.4 or greater, the `ScrollView`, `VStack`
-    /// and `ForEach` can be replaced with a `List` with `scrollContentBackground(.hidden)` applied.
-    private var list: some View {
-        ScrollView {
-            VStack(alignment: .leading) {
-                ForEach(sortedBookmarks, id: \.self) { bookmark in
-                    Button {
-                        selectedBookmark = bookmark
-                    } label: {
-                        Text(bookmark.name)
-                            // Make the entire row tappable.
-                            .frame(maxWidth: .infinity, alignment: .leading)
-                            .contentShape(Rectangle())
-                    }
-                    .buttonStyle(.plain)
-                    .padding(4)
-                    if bookmark != sortedBookmarks.last {
-                        Divider()
-                    }
+    /// Makes a view that shows a list of bookmarks.
+    /// - Parameter bookmarks: The bookmarks to be shown.
+    @ViewBuilder private func makeList(bookmarks: [Bookmark]) -> some View {
+        if bookmarks.isEmpty {
+            noBookmarks
+        } else {
+            List(bookmarks.sorted { $0.name <  $1.name }, id: \.self, selection: selection) { bookmark in
+                Button {
+                    selectBookmark(bookmark)
+                } label: {
+                    Text(bookmark.name)
+                        // Make the entire row tappable.
+                        .frame(maxWidth: .infinity, alignment: .leading)
+                        .contentShape(Rectangle())
                 }
+                .buttonStyle(.plain)
+#if targetEnvironment(macCatalyst)
+                .listRowBackground(bookmark == selection?.wrappedValue ? nil : Color.clear)
+#endif
             }
+            .frame(idealWidth: 320, idealHeight: 428)
```

Comment:
Samples makes [regular use](https://github.com/search?q=repo%3AEsri%2Farcgis-maps-sdk-swift-samples+.frame%28idealWidth%3A+320%2C+idealHeight%3A+428%29&type=code) of this ratio.